akka-docs/src/main/paradox/typed/from-classic.md

```diff
@@ -329,8 +332,8 @@ Retrieving the children from the `ActorContext` can still be useful for some cer
 
 Starting an actor on a remote node—so called remote deployment—isn't supported in Typed.
 
-The main reason is that we would discourage this feature since it often result too tight coupling
-between nodes and undesired failure handling. For example if the node of the parent actor crashes
+The main reason is that we would discourage this feature since it often results in a too tight coupling
+between nodes and failure handling. For example if the node of the parent actor crashes,
```

Comment:
```suggestion
This feature would be discouraged because it often results in tight coupling between nodes and undesirable failure handling. For example if the parent actor's node crashes,
```

Reply:
I new understood what _**undesirable** failure handling_ means... I guess it's the _handling_ that's undesirable, but I read it as _undesirable failure_.
Can you clarify?

Reply:
I added your suggestion in [317dd1c](https://github.com/akka/akka/pull/28452/commits/317dd1c4432287f02e242947534c9c8cc5eba4ee)